packages/aggregator/src/analytics/aggregateAnalytics/aggregateAnalytics.js

```diff
@@ -48,7 +48,7 @@ export const aggregateAnalytics = (
         analytics,
         {
           ...aggregationConfig,
-          fillEmptyPeriodsTilNow: true,
+          fillEmptyPeriodsWith: 'previous',
```

Comment:
Can we flip the order of these so that the config can override the default of `previous`